src/index.ts

```diff
@@ -213,10 +213,11 @@ export const requestPurchase = (
         if (!('sku' in request)) {
           throw new Error('sku is required for iOS purchase');
         }
-        const {sku, appAccountToken, quantity, withOffer} = request;
+        const {sku, autoFinish, appAccountToken, quantity, withOffer} = request;
         const offer = offerToRecordIos(withOffer);
         const purchase = await ExpoIapModule.buyProduct(
           sku,
+          autoFinish ?? true,
```

Comment:
should be `false` in default!
Related https://github.com/hyochan/react-native-iap/blob/0cf6311a48c6087d39b5502792eae7fc5f80c91a/src/iap.ts#L595C1-L596C1